examples/basic_example.yaml

```diff
@@ -40,6 +40,7 @@ pages:
    content:
     - container: TablePlotter
       csv_file: ./example_data.csv
+      filter_cols: ['Well', 'Segment', 'Average permeability (D)']
```

Comment:
Is this the best way, or should we demonstrate the other way of defining lists in yaml (i.e. one col per line). Verbosity vs «less-code-looking» config.